pkg/middleware/auth_test.go

```diff
@@ -11,7 +11,7 @@ func TestMiddlewareAuth(t *testing.T) {
 	Convey("Given the grafana middleware", t, func() {
 		reqSignIn := Auth(&AuthOptions{ReqSignedIn: true})
 
-		middlewareScenario("ReqSignIn true and unauthenticated request", func(sc *scenarioContext) {
+		middlewareScenario(t, "ReqSignIn true and unauthenticated request", func(sc *scenarioContext) {
```

Comment:
Had to change the signature of that method (and `recoveryScenario`) and therefore every instance of its use, seems fine, but if there is better method... :)

Reply:
We can put remotecache in the request context, but it seems a bit more cleaner via DI